servicetalk-opentracing-inmemory/src/main/java/io/servicetalk/opentracing/inmemory/KeyValueFormatter.java

```diff
@@ -31,15 +29,16 @@
 import static io.servicetalk.opentracing.internal.HexUtils.validateHexBytes;
 import static java.lang.String.valueOf;
 
-final class TracingHttpHeadersFormatter implements InMemorySpanContextFormat<HttpHeaders> {
-    private static final Logger logger = LoggerFactory.getLogger(TracingHttpHeadersFormatter.class);
+public final class KeyValueFormatter implements InMemorySpanContextFormat<CharSequenceKeyValueAccessor> {
```

Comment:
Since were breaking the name always, it might a good idea to throw the name `zipkin` or `b3` in there since this is formatter the trace context in b3/zipkin format. 

another suggestion is that we somehow provided a way for users to control the key names.

Reply:
Agreed on 'B3'. 
Regarding the keys I don't want to relax the API yet inside ST especially since this is now B3 compliant. The possibility of altering the names can still happen through the carrier set/get methods (overriding the keys).